implementations/index.md

```diff
@@ -114,6 +114,8 @@ has a page describing how to emit conformant JSON.
 
 * [Migrap.AspNet.Mvc.Jsonapi](https://github.com/migrap/Migrap.AspNet.Mvc.Jsonapi) is an ASP.NET 5 (vNext) library that allows for existing code to build JSON API responses through output formatters.
 
+* [Saule](https://github.com/joukevandermaas/saule/) is a small Json Api 1.0 compatible library that integrates well with established Web Api conventions. It has complete documentation and near 100% test coverage.
```

Comment:
Could you please change "Json Api" -> "JSON API" and "Web Api" -> "Web API"? Thanks in advance.